.packit.yaml

```
@@ -9,6 +9,20 @@ files_to_sync:
   - README.md
   - .packit.yaml
   - gating.yaml
+  - ci.fmf
+# we need to sync tmt for TF to work in Fedora updates
+  - src: ./tests/smoke/*
+    dest: ./tests/smoke/
+  - src: ./plans*
```

```suggestion
  - src: ./plans/*
```
can't tell if the former would work

---

thanks!

---

fixed